src/backend/routers/auth.py

```diff
@@ -317,3 +319,68 @@ def log_and_redirect_err(error_message: str):
     response = RedirectResponse(redirect_uri)
 
     return response
+
+
+@router.delete("/tool/auth/{tool_id}")
+async def delete_tool_auth(
```

Comment:
Would be awesome to have some tests

Reply:
Thank you for the reminder! I added unit tests for tool auth crud, and I will make a follow-up PR with integration tests, and tag you and @lusmoura!